storagenode/storagenodedb/database.go

```diff
@@ -699,6 +699,29 @@ func (db *DB) Migration() *migrate.Migration {
 					)`,
 				},
 			},
+			{
+				DB:          db.versionsDB,
+				Description: "Create satellites table and satellites_exit_progress table",
+				Version:     21,
+				Action: migrate.SQL{
+					`CREATE TABLE satellites (
+						node_id BLOB NOT NULL,
+						address TEXT NOT NUll,
+						added_at TIMESTAMP NOT NULL,
+						status INTEGER NOT NULL,
+						PRIMARY KEY (node_id)
+					)`,
+					`CREATE TABLE satellites_exit_progress (
```

Comment:
I know the blueprint called it `satellites_exit_progress`, but I think we should change to `satellite_exit_progress`

Can you update the blueprint in this PR with the `status` and table name changes.  It would be good to track them together I think.

Reply:
updated it